lib/iris/coords.py

```diff
@@ -2816,25 +2816,24 @@ def __init__(self, method, coords=None, intervals=None, comments=None):
         self._init(method, tuple(_coords), tuple(_intervals), tuple(_comments))
 
     def __str__(self):
```

Comment:
As mentioned in my `cube_summary.py` comment: could you abstract out the code that generates the method + interval + comment string (e.g. `mean (6 minutes, This is a test comment, 12 minutes)`) into its own method: `self.method_str`?

Then just have `self.__str__` stitch together `self.coord_names` and `self.method_str()`? That would avoid the need for downstream string parsing, which can be risky and difficult to follow.